### PR TITLE
fix(mixins): add backend path section in loki-operational for single scalable deployment

### PR DESCRIPTION
**Pull request by QuentinBisson**
**What this PR does / why we need it**:

This PR adds a backend path row in loki operational just after the read path

**Which issue(s) this PR fixes**:
Fixes #<issue number>

**Special notes for your reviewer**:

**Checklist**
- [x] Reviewed the [`CONTRIBUTING.md`](https://github.com/grafana/loki/blob/main/CONTRIBUTING.md) guide (**required**)
- [ ] Documentation added
- [ ] Tests updated
- [x] Title matches the required conventional commits format, see [here](https://www.conventionalcommits.org/en/v1.0.0/)
  - **Note** that Promtail is considered to be feature complete, and future development for logs collection will be in [Grafana Alloy](https://github.com/grafana/alloy). As such, `feat` PRs are unlikely to be accepted unless a case can be made for the feature actually being a bug fix to existing behavior.
- [ ] Changes that require user attention or interaction to upgrade are documented in `docs/sources/setup/upgrade/_index.md`
- [ ] For Helm chart changes bump the Helm chart version in `production/helm/loki/Chart.yaml` and update `production/helm/loki/CHANGELOG.md` and `production/helm/loki/README.md`. [Example PR](https://github.com/grafana/loki/commit/d10549e3ece02120974929894ee333d07755d213)
- [ ] If the change is deprecating or removing a configuration option, update the `deprecated-config.yaml` and `deleted-config.yaml` files respectively in the `tools/deprecated-config-checker` directory. [Example PR](https://github.com/grafana/loki/pull/10840/commits/0d4416a4b03739583349934b96f272fb4f685d15)

### CODE REVIEW

**Comment by cstyan** [comment]
sorry for the delay @QuentinBisson, I was away last week, can review this week

**Comment by QuentinBisson** [comment]
@cstyan don't worry about it, I was busy fixing Mimir mixins 😅

Now regarding this one, i was wondering if it would not be better to have a single single scalable resource dashboard instead of having 1 for read, 1 for write and 1 for backend that all contain only 1 row anyway. What do you think?

**Comment by cstyan** [comment]
> @cstyan don't worry about it, I was busy fixing Mimir mixins 😅
> 
> Now regarding this one, i was wondering if it would not be better to have a single single scalable resource dashboard instead of having 1 for read, 1 for write and 1 for backend that all contain only 1 row anyway. What do you think?

that seems like a good idea to me :+1:

**Comment by QuentinBisson** [comment]
@cstyan I renamed the PR to match what it really does as the Merge PR for resources is here https://github.com/grafana/loki/pull/13471

**Comment by QuentinBisson** [comment]
@cstyan would you be able to review my PRs?

**Comment by cstyan** [comment]
@QuentinBisson again, need to merge in main after the other PRs were merged

File: production/loki-mixin/dashboards.libsonnet [hunk 1]
```diff
@@ -4,9 +4,9 @@
 (import 'dashboards/loki-logs.libsonnet') +
 (import 'dashboards/loki-operational.libsonnet') +
 (import 'dashboards/loki-reads.libsonnet') +
+(import 'dashboards/loki-reads-resources.libsonnet') +
```
Comment (cstyan) [inline production/loki-mixin/dashboards.libsonnet:7]: did you mean to change this, or was it a linter change?

Comment (QuentinBisson) [reply production/loki-mixin/dashboards.libsonnet:7]: I did, I thought having both read dashboards close to each other made sense but I can revert this one of you prefer

**Review by cstyan — APPROVED** [review]

**Pull request merged**